common/mb_controller.c

```diff
@@ -186,6 +188,18 @@ int mb_load_controller_config(pid_parameters_t* pid_params){
     return 0;
 }
 
+// Reset all the filters used in the controller
+// Return 0 on success
+int mb_controller_filter_reset(mb_state_t* mb_state, mb_setpoints_t* mb_setpoints){
+
+    rc_filter_reset(&pid_filt_l);
+    rc_filter_reset(&pid_filt_r);
+    rc_filter_reset(&lp_filt_l);
+    rc_filter_reset(&lp_filt_r);
```

Comment:
Can you wrap these in an if statement as well?

Reply:
Yup, I'll do it with the destroy function too.